src/Solver/Interoperability.cpp

```diff
@@ -896,10 +886,33 @@ seissol::Interoperability::initializeIO(double* mu, double* slipRate1, double* s
   
 	// Initialize checkpointing
 	int faultTimeStep;
-	bool hasCheckpoint = seissol::SeisSol::main.checkPointManager().init(reinterpret_cast<real*>(m_ltsTree->var(m_lts->dofs)),
+
+  // Only R&S friction explicitly stores the state variable, otherwise use the accumulated slip magnitude
```

Comment:
why not 0?
In the end, accumulatedSlipMagnitude will be stored anyway.

Reply:
Then, I would need to create an array with a lot of zeros first ;-)

Reply:
Ok, in the end, we would need some more flexibility for checkpointing, e.g. with TP T and P should be added, etc.
Let's leave it like that now, or add a TODO